GUI/Types/Renderer/Shaders/simple.frag

```diff
@@ -86,14 +86,14 @@ void main()
 #endif
 
 #if param_renderMode_Normals
-	outputColor = vec4(vNormalOut, 1.0);
+	outputColor = vec4(vNormalOut * vec3(0.5) + vec3(0.5), 1.0);
```

Comment:
All fragment shaders should visualize normals and bump normals the same. So either revert this change or update the renderMode code in the other fragment shaders too. I personally prefer keeping it the way it was.

Reply:
As it was before, all negative values would be just black. It half the spectrum. I'll update the rest.